client/src/components/mapping/FhirMappingPanel/FhirResourceTree/index.tsx

```diff
@@ -247,14 +247,15 @@ const FhirResourceTree = ({ onClickCallback }: Props) => {
     attributes.map(buildNodeFromAttribute).filter(Boolean) as TreeNode[];
 
   const fetchAttributeDefinition = async (
-    definitionId: string
+    definitionId: string,
+    parent?: Attribute
```

Comment:
should this be optional ?

Reply:
do you think it's best to pass only the parent, and use parent.types[0] as the definitionId ?

Reply:
You're right! I can do that 👍

Reply:
Done